### PR TITLE
fix: typos

### PR DESCRIPTION
**Pull request by omahs**
fix: typos

### CODE REVIEW

File: examples/demo-rollup/README.md [hunk 1]
```diff
@@ -208,7 +208,7 @@ pub enum CallMessage<C: sov_modules_api::Context> {
 }
 ```
 
-In the above snippet, we can see that `CallMessage` in `Bank` support five different types of calls. The `sov-cli` has the ability to parse a JSON file that aligns with any of these calls and subsequently serialize them. The structure of the JSON file, which represents the call, closely mirrors that of the Enum member. Consider the `Transfer` message as an example:
+In the above snippet, we can see that `CallMessage` in `Bank` supports five different types of calls. The `sov-cli` has the ability to parse a JSON file that aligns with any of these calls and subsequently serialize them. The structure of the JSON file, which represents the call, closely mirrors that of the Enum member. Consider the `Transfer` message as an example:
```
Comment (neysofu) [inline examples/demo-rollup/README.md:211]: The original was correct in this case, let's revert this.

File: examples/demo-rollup/benches/README.md [hunk 2]
```diff
@@ -71,4 +71,4 @@ The Makefile is located in the demo-rollup/benches folder and supports the follo
 
 The Makefile supports setting number of blocks and transactions per block using BLOCKS and TXNS_PER_BLOCK env vars. Defaults are 100 blocks and 10,000 transactions per block when using the Makefile
 
-![Flamgraph](flamegraph_sample.svg)
\ No newline at end of file
+![Flamgraph](flamegraph_sample.svg)
```
Comment (neysofu) [inline examples/demo-rollup/benches/README.md]: While we're at it, let's fix this one as well:

```suggestion
![Flamegraph](flamegraph_sample.svg)
```

**Review by neysofu — CHANGES_REQUESTED** [review]
Thanks a bunch for all these fixes! Greatly appreciated.

I provided ~two suggestions~ one suggestion below :smile:

File: examples/demo-rollup/README.md (hunk 1 above)
Comment (neysofu) [reply examples/demo-rollup/README.md:211]: Nevermind, I got confused while reviewing :P

**Review by neysofu — APPROVED** [review]

**Review by bkolad — APPROVED** [review]

**Pull request merged**